MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
@tgomudur  and @super9user I have created new branch with following pull request #48